src/store/FilingHistoryList/getters.ts

```diff
@@ -77,5 +77,24 @@ export default {
   /** Whether one document is downloading. */
   isLoadingOne (state: FilingHistoryListStateIF): boolean {
     return state.loadingOne
+  },
+
+  /** Whether the business is authorized to continue out, i.e. true if cco expiry date is present or in the future. */
+  isAuthorizedToContinueOut (state: FilingHistoryListStateIF, rootGetters: any): boolean {
+    const ccoFiling = state.filings.find(val => {
+      const exp = val.data?.consentContinuationOut?.expiry
+      if (exp) {
+        return true
+      }
+      return false
+    })
```

Comment:
No change needed, but if you wanted to simplify this a bunch then you could write:
```
const ccoFiling = state.filings.find(f => f.data?.consentContinuationOut?.expiry)
```

Actually I think that ^^ is a bit too terse. My guidance is, "is the code easy to read and understand?". Your code is; mine isn't 😁